external/upstream/pcmsolver/CMakeLists.txt

```diff
@@ -1,26 +1,27 @@
 if(${ENABLE_PCMSolver})
-    find_package(PCMSolver QUIET)  # todo CONFIG
+    find_package(PCMSolver CONFIG QUIET)
 
     if(${PCMSolver_FOUND})
-        #get_property(_loc TARGET PCMSolver::pcm PROPERTY LOCATION)  # todo
-        get_property(_loc TARGET PCMSolver::PCMSolver PROPERTY LOCATION)
+        get_property(_loc TARGET PCMSolver::pcm PROPERTY LOCATION)
         message(STATUS "${Cyan}Found PCMSolver${ColourReset}: ${_loc} (found version ${PCMSolver_VERSION})")
         add_library(pcmsolver_external INTERFACE)  # dummy
     else()
         include(ExternalProject)
         message(STATUS "Suitable PCMSolver could not be located, ${Magenta}Building PCMSolver${ColourReset} instead.")
 
         if(${BUILD_SHARED_LIBS})
```

Comment:
The AddOns currently build so/a according to the value of `BUILD_SHARED_LIBS`, which I usually set as ON, but the default is OFF. Fortunately for you, the one that I _must_ have working is the shared. If static is no good with PCMSolver, I'd comment out [this](https://github.com/loriab/psi4/blob/25e2e4347953ae027b0176059cffdf8637b63705/external/upstream/pcmsolver/CMakeLists.txt#L37-L38) and just add a new line `SHARED_LIBRARY_ONLY=ON` as a stopgap.